enzyme/test/Enzyme/ForwardMode/fneg.ll

```diff
@@ -0,0 +1,32 @@
+; RUN: %opt < %s %loadEnzyme -enzyme -enzyme-preopt=false -mem2reg -instsimplify -simplifycfg -S | FileCheck %s
```

Comment:
I think you're going to need to set a check for llvm version above a certain number here.

Reply:
done